src/pages/agent/tests-to-run/tests-to-run.tsx

```diff
@@ -39,6 +40,71 @@ interface Props {
   agentType?: string;
 }
 
+const columnsForTable = [
```

Comment:
```suggestion
const columns = [
```
We inside table component. It means that columns refer to the table